docs/reference/release-notes/7.9.asciidoc

```diff
@@ -29,6 +29,23 @@ find out about the issue after upgrading then reindexing is required to recover.
 Full details of the mitigations are in
 {ml-docs}/ml-troubleshooting.html#ml-troubleshooting-mappings[Upgrade to 7.9.0 causes incorrect mappings].
 
+* Lucene 8.6.0, on which Elasticsearch 7.9.0 is based,
+  https://issues.apache.org/jira/browse/LUCENE-9478[contains a memory
+  leak]. This memory leak manifests in Elasticsearch when a single document is
+  updated repeatedly with a forced refresh. The cluster state storage layer in
+  Elasticsearch is based on Lucene and does use single-document updates with
+  forced refreshes, meaning that this memory leak manifests in Elasticsearch under
```

Comment:
We use `flush` not `refresh` in the cluster state storage. But I think it's okay to leave it as is.